presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -804,6 +804,82 @@ The following query will fail with the error ``Column 'name' is ambiguous``::
     FROM nation
     CROSS JOIN region;
 
+
+USING
+^^^^^
+The ``USING`` clause allows you to write shorter queries when both tables you 
+are joining have the same name for the join key.
+
+For example:
+
+.. code-block:: none 
+
+    SELECT *
```

Comment:
Could you use tpch tables as examples like other examples do?

Reply:
I'm not sure what you mean here with "tpch tables". Can you help direct me to what it is you are referring to? (As far as I can see, other places in the doc uses VALUES for examples that can be ran, and the pseudo code I'm using at the beginning of the section, I feel, is better than a real example)

Reply:
You could start `TpchQueryRunner` and use the tables in side that catalog.
After that you can query against tpch tables
```
cd presto/presto-cli/target
presto-cli-*-executable.jar --catalog tpch --schema sf10
```

All the examples we use are from tpch; so users can directly copy and paste a query on that webpage and run it from cli.

Reply:
Got it.
Thanks for the suggestion, but I suspect this would only make the example more cumbersome. 
The first examples I wrote are not to be run at the terminal, but this trick has been used throughout the text (e.g.: in the "WITH Clause" section), and is (IMHO) a better way to explain the structure.

The last code I added is self contained (and based on VALUES) and the user can run it in the terminal to see how it behaves (this was also used in the text, e.g.: the sections with union, intersect, UNNEST, etc.).

In short, I don't think adding another example would be helpful to the text (and I think this diff is probably good to go as is).

If you have a specific example you think would be helpful, please share it so that I'd better understand what you had in mind.

What do you think?